docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,98 @@
+Policy for Unofficial and Outdated Projects
+===========================================
+
+This policy describes a process where we take actions against unmaintained and unofficial forks of project documentation.
```

Comment:
Should we specify that this is about (software) projects with versioned release cycles?

There are types of documentation projects, especially single version projects, that are a tutorial, a book etc.

Reply:
As long as the policy enforces "reachability" and only works "under requests", I think it's fine.

Also, I liked that Eric added: "The core team does not have any additional reservations.", to one of the points that must happen, so we are covering these cases there as well.